omf-python/src/omf1/converter.rs

```diff
@@ -1,13 +1,43 @@
-use omf::omf1::detect_open;
+use omf::omf1::detect_open as omf1_detect_open;
+use omf::omf1::Converter;
 use pyo3::exceptions::PyIOError;
 use pyo3::prelude::*;
 use std::path::Path;
 
 #[pyfunction]
-pub fn is_omf1(path: String) -> PyResult<bool> {
+pub fn detect_open(path: String) -> PyResult<bool> {
     let path = Path::new(&path);
-    match detect_open(path) {
+    match omf1_detect_open(path) {
         Ok(result) => Ok(result),
         Err(e) => Err(PyErr::new::<PyIOError, _>(e.to_string())),
     }
 }
+
+#[pyclass(name = "Converter")]
+pub struct PyConverter {
+    inner: Converter,
+}
+
+#[pymethods]
+impl PyConverter {
+    #[new]
+    pub fn new() -> PyResult<Self> {
+        let inner = Converter::new();
+        Ok(PyConverter { inner })
+    }
+
+    fn convert_open(&self, input_path: String, output_path: String) -> PyResult<bool> {
+        // TODO: handle errors
+        let problems = self
+            .inner
+            .convert_open(input_path, output_path)
+            .map_err(|e| PyErr::new::<PyIOError, _>(e.to_string()))?;
+
+        // TODO: return problems ?
+        if !problems.is_empty() {
+            println!("Warnings converting: {:?}", problems);
```

Comment:
@antbrown I wonder how we should handle this ... I guess the function could return a vec of PyProblem ? or maybe we just leave it like this and the function could return nothing. maybe it should print to stderr though (eprintln).

Reply:
I guess it's not too difficult I probably should return them, and we should probably just print them to stderr in our data converters library to help debug issues